dashboards/google-kubernetes-engine/gke-workloads-at-risk.json

```diff
@@ -0,0 +1,365 @@
+                }
+              }
+            ],
+            "metricVisualization": "NUMBER"
+          },
+          "title": "Memory (reliability risk) - workloads running above requested for memory"
+        },
+        "width": 6,
+        "xPos": 0,
+        "yPos": 10
+      },
+      {
+        "height": 6,
+        "widget": {
+          "collapsibleGroup": {
+            "collapsed": false
+          },
+          "title": "14-day Peak Recommendations (Deployments only)"
+        },
+        "width": 12,
+        "xPos": 0,
+        "yPos": 14
+      },
+      {
+        "height": 4,
+        "widget": {
+          "timeSeriesTable": {
+            "dataSets": [
+              {
+                "timeSeriesQuery": {
+                  "timeSeriesQueryLanguage": "fetch k8s_scale\n| metric\n    'kubernetes.io/autoscaler/container/memory/per_replica_recommended_request_bytes'\n| group_by 14d,\n    [value_per_replica_recommended_request_bytes_mean:\n       max(value.per_replica_recommended_request_bytes)]\n| every 5m\n\n"
+                }
+              }
+            ],
+            "metricVisualization": "NUMBER"
+          },
+          "title": "MEM Recommended per replica request bytes [MAX]"
+        },
+        "width": 6,
+        "xPos": 0,
+        "yPos": 16
+      },
+      {
+        "height": 4,
+        "widget": {
+          "timeSeriesTable": {
+            "dataSets": [
+              {
+                "timeSeriesQuery": {
+                  "timeSeriesQueryLanguage": "fetch k8s_scale\n| metric\n    'kubernetes.io/autoscaler/container/cpu/per_replica_recommended_request_cores'\n| group_by 14d,\n    [value_per_replica_recommended_request_cores_mean:\n       max(value.per_replica_recommended_request_cores)]\n| every 5m"
+                }
+              }
+            ],
+            "metricVisualization": "NUMBER"
+          },
+          "title": "CPU Recommended per replica request cores [MAX] "
+        },
+        "width": 6,
+        "xPos": 6,
+        "yPos": 16
+      },
+      {
+        "height": 2,
+        "widget": {
+          "text": {
+            "content": "Recommended resource requests for CPU and memory requirements, ensuring efficient resource based on VPA recommendations.\n\n**Want to see workload recommendations at scale?**\n[Right-size your GKE workloads](https://cloud.google.com/blog/products/containers-kubernetes/optimize-gke-workloads-at-scale)",
```

Comment:
How about something like 

> Refer to the following recommended resource requests for CPU and memory requirements that ensuring efficient resource allocation based on VPA recommendations. They are calculated as 14 day peak recommendations.
> 
> **Want to see workload recommendations at scale?** [Right-size your GKE workloads](https://cloud.google.com/blog/products/containers-kubernetes/optimize-gke-workloads-at-scale)

and no title, if we've moved the title up into the collapsible group

Reply:
updated